dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/repository/BrowserRepository.java

```diff
@@ -169,8 +168,8 @@ private static QueryBuilder getMutationFilter(String segmentId, Long start, Long
       List<String> projectFilters, List<String> impacts) {
     val andQueryFilter = boolQuery()
         .must(termQuery("chromosome", segmentId))
-        .must(rangeQuery("chromosome_start").lte(stop))
-        .must(rangeQuery("chromosome_end").lte(start));
+        .must(rangeQuery("chromosome_start").gte(start).lte(stop))
```

Comment:
Is this right? Don't we want to show things that overlap, not just things that are strictly contained?